docs/how-to/deploy_sdcore_user_plane_in_dpdk_mode.md

```diff
@@ -0,0 +1,125 @@
+# Deploy SD-Core User Plane in DPDK mode
+
+This guide covers how to deploy the User Plane Function (UPF) in DPDK mode using 
+the `sdcore-user-plane` Juju bundle.
+
+## Requirements
+
+- A Kubernetes cluster which meets below requirements:
+  - host CPU that supports AVX2, RDRAND and PDPE1GB instructions (Intel Haswell, AMD Excavator 
+    or equivalent)
+  - SR-IOV interfaces for Access and Core networks
+  - At least two 1G HugePages available
+  - `driverctl` installed
+  - LoadBalancer with 1 available address for the UPF
+  - Multus CNI enabled
+- Juju >= 3.1/stable
+- A Juju controller bootstrapped onto the Kubernetes cluster
+- Juju model created
+
+## Changing the driver of the network interfaces to `vfio-pci`
+
+As `root` user, load the `vfio-pci` driver on the Kubernetes host:
+
+```shell
+echo "vfio-pci" > /etc/modules-load.d/vfio-pci.conf
+modprobe vfio-pci
+```
+
+```{note}
+Using `vfio-pci`, by default, needs IOMMU to be enabled. In the environments which do not support
+IOMMU, `vfio-pci` needs to be loaded with additional module parameter:
+`echo "options vfio enable_unsafe_noiommu_mode=1" > /etc/modprobe.d/vfio-noiommu.conf`
+```
+
+Get PCI address of `access` and `core` interfaces:
+
+```shell
+$ sudo lshw -c network -businfo
+Bus info          Device           Class      Description
+=========================================================
+pci@0000:00:05.0  ens5             network    Elastic Network Adapter (ENA)
+pci@0000:00:06.0  ens6             network    Elastic Network Adapter (ENA) # access interface
+pci@0000:00:07.0  ens7             network    Elastic Network Adapter (ENA) # core interface
+```
+
+Bind `access` and `core` interfaces to the `vfio-pci` driver:
+
+```shell
+sudo driverctl set-override 0000:00:06.0 vfio-pci
+sudo driverctl set-override 0000:00:07.0 vfio-pci
+```
+
+## Configuring Kubernetes plumbing required by DPDK
```

Comment:
```suggestion
## Create the SR-IOV configuration file
```

Reply:
This was marked as "resolved" but it's not

Reply:
Ups, sorry, too much automation. 
And actually this paragraph is much more then just creation of the configuration file. In the next step, we deploy a DaemonSet which uses this config. This part makes K8s aware of our network interfaces.
Next we add a `vfio-veth` CNI, which allows bessd to utilize the network resources created above.

Reply:
Ok I just updated my suggestion

Reply:
Applied.